src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/opensuse/42.3/Microsoft.NETCore.Runtime.CoreCLR.pkgproj

```diff
@@ -14,7 +14,6 @@
     <NativeSplittableBinary Include="$(BinDir)libmscordaccore.so" />
     <NativeSplittableBinary Include="$(BinDir)libmscordbi.so" />
     <NativeSplittableBinary Include="$(BinDir)libsos.so" />
-    <NativeSplittableBinary Include="$(BinDir)libsosplugin.so" />
```

Comment:
I wonder - should we do this and surprise users by missing plugin or should we instead build the missing liblldb during the docker image creation? I have a list of commands to build lldb 3.9, so maybe it would work for the lldb 3.8 too.

Reply:
There is an liblldb package but it doesn't include the actual header files that we need to build the plugin. Does you method create those headers somehow? Or does it pull them from some unsupported package source?

Reply:
The headers come from the source code. Building the library also produces the headers.

Reply:
My main reason I left out the libsosplugin because the package with the headers wasn't available as a supported download from opensuse for 42.3. If we're willing to go with a different route, then why don't we use the 15.0 package for liblldb? It's not officially supported for 423, but it has what we need and it works. I've already got a dockerfile ready for it:

e.g.
`
RUN zypper ar -f http://download.opensuse.org/tumbleweed/repo/oss/suse/ TumbleweedOSS
RUN zypper -n install lldb5-devel
`

Reply:
We just need to make sure the liblldb version matches the lldb version we expect users to have on their 423 machines.

Reply:
I imagine that would be preferable to leaving out the plugin entirely.

Reply:
As long as it doesn't prevent the user from using lldb3.8 should they not care about the exclusion of the libsos plugin

Reply:
Can you try to install the lldb 5.0 and verify that it doesn't uninstall the 3.8? I've seen this happen on some Linux distros while they could coexist just fine on another one.

Reply:
Installing 5.0 conflicts with 3.8..

Reply:
It also causes my bootstrap CLI for opensuse to fail.

Reply:
Hmm, I was afraid that would be the case. So it seems we either need to build the library while creating the docker image or drop the plugin.

Reply:
If we build the library while creating the image, would that mean that users would need to also build the library to use the plugin we create?

Reply:
Let me check if the runtime library is part of the lldb package or not.

Reply:
It should just work, the runtime liblldb.so.3.8 is part of the lldb package.

Reply:
In that case, that seems the best route. Do you want to make the change to the opensuse dockerfile yourself or provide me with the list of commands to pull sources and build them manually?

Reply:
I'll update the dockerfile, I will be testing the change in docker anyways, so it seems like the most straightforward path.

Reply:
Sounds good, thanks Jan. I'll keep this PR open as a fallback in case the manual build doesn't work out for 3.8.